python/tests/test_account.py

```diff
@@ -823,32 +834,27 @@ def test_send_and_receive_message_markseen(self, acfactory, lp):
         ac1.set_config("displayname", "ä name")
 
         lp.sec("ac1: create chat with ac2")
-        chat = get_chat(ac1, ac2)
+        chat = acfactory.get_chat(ac1, ac2)
 
         lp.sec("sending text message from ac1 to ac2")
-        msg_out = chat.send_text("message1")
-        ev = ac1._evtracker.get_matching("DC_EVENT_MSG_DELIVERED")
-        assert ev.data1 == chat.id
-        assert ev.data2 == msg_out.id
-        assert msg_out.is_out_delivered()
+        msg1 = chat.send_text("message1")
+        ac1._evtracker.wait_msg_delivered(msg1)
 
         lp.sec("wait for ac2 to receive message")
-        ev = ac2._evtracker.get_matching("DC_EVENT_MSGS_CHANGED")
-        assert ev.data2 == msg_out.id
-        msg_in = ac2.get_message_by_id(msg_out.id)
-        assert msg_in.text == "message1"
-        assert not msg_in.is_forwarded()
-        assert msg_in.get_sender_contact().display_name == ac1.get_config("displayname")
+        msg2 = ac2._evtracker.wait_next_messages_changed()
```

Comment:
Sometimes `get_matching("DC_EVENT_MSGS_CHANGED")` is used, sometimes `_evtracker.get_matching("DC_EVENT_INCOMING_MSG")`. This always was this way (well, since I am part of the DC team) but maybe we should change this one day. Or is there a reason I do not see?

Reply:
INCOMING_MSG comes when the incoming sender is accepted already. 
DC_EVENT_MSGS_CHANGED is called for new deaddrop messages.